### PR TITLE
Updated the form validation, edit form and added bootstrap design cha…

### PR DESCRIPTION
**Pull request by elangovansundar**
…nges

### CODE REVIEW

File: src/Controller/UserController.php [hunk 1]
```diff
@@ -35,35 +37,43 @@ public function index()
...
-                'constraints' => new NotBlank(),
+                'constraints' => new NotBlank(), 'attr' => array('class' => 'form-control')
             ))
             ->add('userLastName', TextType::class, array(
-                'constraints' => new NotBlank(),
+                'constraints' => new NotBlank(), 'attr' => array('class' => 'form-control')
             ))
             ->add('userEmail', TextType::class, array(
-                'constraints' => new NotBlank(),
+                'constraints' => new NotBlank(), 'attr' => array('class' => 'form-control')
```
Comment (aspire-shatheesh) [inline src/Controller/UserController.php]: all form validations has to be separate..

Comment (aspire-shatheesh) [reply src/Controller/UserController.php]: i.e validations should be in yml file

File: src/Controller/UserController.php [hunk 2]
```diff
@@ -35,35 +37,43 @@ public function index()
...
         $form = $this->createFormBuilder()
             ->add('userFirstName', TextType::class, array(
-                'constraints' => new NotBlank(),
+                'constraints' => new NotBlank(), 'attr' => array('class' => 'form-control')
             ))
             ->add('userLastName', TextType::class, array(
-                'constraints' => new NotBlank(),
+                'constraints' => new NotBlank(), 'attr' => array('class' => 'form-control')
             ))
             ->add('userEmail', TextType::class, array(
```
Comment (aspire-shatheesh) [inline src/Controller/UserController.php]: did the requirement was to have multiple emails?

File: src/Controller/UserController.php [hunk 3]
```diff
@@ -107,6 +121,119 @@ public function getManageUsersAction()
...
+                    'constraints' => new NotBlank(), 'attr' => array('class' => 'form-control'), 'data' => $users->getUserLastName()
+                ))
+                ->add('userEmail', TextType::class, array(
+                    'constraints' => new NotBlank(), 'attr' => array('class' => 'form-control'), 'data' => $userEmailString
+                ))
+                ->add('userMobileNumber', TextType::class, array(
+                    'constraints' => new NotBlank(), 'attr' => array('class' => 'form-control'), 'data' => $userMobileString
+                ))
+                ->add('userDateofBirth', TextType::class, array(
+                    'constraints' => new NotBlank(), 'attr' => array('class' => 'form-control'), 'data' => $users->getUserDateofBirth()
```
Comment (aspire-shatheesh) [inline src/Controller/UserController.php]: these are not recommended approaches in symfony.

File: src/Controller/UserController.php [hunk 4]
```diff
@@ -107,6 +121,119 @@ public function getManageUsersAction()
...
+                $data = $form->getData();
+
+                $dm = $this->get('doctrine_mongodb')->getManager();
+                $user = $dm->getRepository(User::class)->find($data->getId());
+
+                $userEmailArray  = explode(",",$data->getUserEmail());
+                $userMobileArray = explode(",",$data->getUserMobileNumber());
+
+                $user->setUserFirstName($data->getUserFirstName());
+                $user->setUserLastName($data->getUserLastName());
```
Comment (aspire-shatheesh) [inline src/Controller/UserController.php]: this is also not a recommended approach in symfony

File: src/Controller/UserController.php (hunk 2 above)
Comment (elangovansundar) [reply src/Controller/UserController.php]: Inthis, We can use multiple emails by using comma separator.

Comment (aspire-shatheesh) [reply src/Controller/UserController.php]: Does it follows / conform to DB Normal Form / DB Normalization

Comment (aspire-shatheesh) [reply src/Controller/UserController.php]: Also in general, in websites, how they accept multiple emails / mails or multiple phone numbers /  as comma separated?

Comment (elangovansundar) [reply src/Controller/UserController.php]: ok i will update.

File: src/Controller/UserController.php [hunk 5]
```diff
@@ -107,6 +121,119 @@ public function getManageUsersAction()
...
+                ->add('userLastName', TextType::class, array(
+                    'constraints' => new NotBlank(), 'attr' => array('class' => 'form-control'), 'data' => $users->getUserLastName()
+                ))
+                ->add('userEmail', TextType::class, array(
+                    'constraints' => new NotBlank(), 'attr' => array('class' => 'form-control'), 'data' => $userEmailString
+                ))
+                ->add('userMobileNumber', TextType::class, array(
+                    'constraints' => new NotBlank(), 'attr' => array('class' => 'form-control'), 'data' => $userMobileString
+                ))
+                ->add('userDateofBirth', TextType::class, array(
```
Comment (aspire-shatheesh) [inline src/Controller/UserController.php]: also is date of birth a text field? how usually websites have it.?

File: src/Controller/UserController.php (hunk 1 above)
Comment (elangovansundar) [reply src/Controller/UserController.php]: I have handled the form validations in yml file.

File: src/Controller/UserController.php (hunk 2 above)
Comment (elangovansundar) [reply src/Controller/UserController.php]: I have handled the multiple email and mobile number as expected.

File: src/Controller/UserController.php (hunk 3 above)
Comment (elangovansundar) [reply src/Controller/UserController.php]: I have modified the form as symfony way  approach.

File: src/Controller/UserController.php (hunk 4 above)
Comment (elangovansundar) [reply src/Controller/UserController.php]: I have modified the symfony way approach.

File: src/Controller/UserController.php (hunk 5 above)
Comment (elangovansundar) [reply src/Controller/UserController.php]: Changed the date of birth functionality by selecting the date from calender.

**Pull request merged**